File: JAVA/17_Number&CombinationTheory/BOJ_07_11050_이항계수1.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {

    public static int fact(int n) {
        if (n <= 1) return 1;
        return n * fact(n - 1);
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");
        int n = Integer.parseInt(st.nextToken());
        int k = Integer.parseInt(st.nextToken());
        System.out.println(fact(n) / (fact(n - k) * fact(k)));
    }
}

File: C/17_Number&CombinationTheory/BOJ_07_11050_이항계수1.c
#include <stdio.h>
int fact(int n) {
	if (n <= 1) return 1;
	else return n * fact(n - 1);
}
main() {
	int n, k;
	scanf("%d %d", &n, &k);
	printf("%d", fact(n) / (fact(n - k) * fact(k)));
}